io_scene_godot/converters/simple_nodes.py

```diff
@@ -115,23 +140,25 @@ def export_lamp_node(escn_file, export_settings, node, parent_gd_node):
 
     light = node.data
 
+    klass = CyclesLightNode if light.use_nodes else LightNode
+
     if light.type == "POINT":
-        light_node = LightNode(node.name, 'OmniLight', parent_gd_node)
+        light_node = klass(node.name, 'OmniLight', parent_gd_node)
 
-        if not light.use_sphere:
+        if not (light.use_sphere or light.use_nodes):
             logging.warning(
                 "Ranged light without sphere enabled: %s", node.name
             )
 
     elif light.type == "SPOT":
-        light_node = LightNode(node.name, 'SpotLight', parent_gd_node)
-        if not light.use_sphere:
+        light_node = klass(node.name, 'SpotLight', parent_gd_node)
+        if not (light.use_sphere or light.use_nodes):
```

Comment:
I prefer here add a line of comment says cycle light has no `use_sphere`, and do you think it is logically more clear to write it as `if not light.use_nodes and not light.use_sphere`?